ExtendedDescription.md

```diff
@@ -0,0 +1,130 @@
+
+Here at nelson.social, we believe in the power of community and the importance of supporting one another in order to enrich our community. Whether you are a lifelong resident of Nelson, new to the region, or just passing through, we invite you to join the conversation and be a part of our community.
+
+In this space, you can share your thoughts and ideas, ask questions of your neighbours, and even plan events and activities. You can also connect with people from all over the world, because nelson.social is part of the [Mastodon](https://github.com/mastodon/) network. We like to think of it like a big WhatsApp or iMessage group, but where you can choose who to share with and whose messages you see, even if they aren’t members of nelson.social.
+
+We encourage you to be respectful and considerate of others, and to use this space to help build a stronger, more connected community in Nelson. We have a simple set of rules for the space that help the team that runs nelson.social ensure that we can be welcoming for everyone. We encourage you to read them, and also reach out to us if you encounter any problems, or have any questions.
+
+We look forward to seeing you on nelson.social, and to building a vibrant and supportive community together. Welcome!
+
+## Land Acknowledgement
+
+We would like to acknowledge that in Nelson we live and work on the traditional, ancestral and unceded territory of Sinixt Peoples. We make this acknowledgment to show our respect for the tmxʷulaʔxʷ (homeland), and [Sinixt](https://bloodoflifecollective.org/sinixt.html) Nation. We feel privileged and grateful to be here, and we invite those reading this to explore their own relationship to place.
+
+## Code of Conduct
+
+Nelson is home to people from all over the world who have a diverse set of skills, personalities, backgrounds and experiences. This diversity is wonderful, and we are committed to ensuring that the nelson.social community is a safe, enjoyable and thriving environment for the largest number possible of Nelson residents, their friends and family members.
+
+To that end, we will strive to make this a friendly, safe and welcoming environment for everyone regardless of qualities such as their gender identity or expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, age, race, nationality, ethnicity, socioeconomic status, family structure, spirituality, religion (or lack thereof), education, or other personal traits. We particularly celebrate diversity and do not tolerate bigotry or prejudice.
+
+As a community, we expect any participant to act according to the following ground rules. They ensure that everyone within the community is clear about the behaviour that is expected of them, and that newcomers know what kind of environment they can expect to find here.
```

Comment:
The reference of "following ground rules", what does it refer to? The next sentence or "guiding principle"?

Reply:
Yeah we should be more explicit and say "guidelines in this document" or some such. Good catch!

Reply:
```suggestion
As a community, we expect any participant to act according to the guidelines in this document. They ensure that everyone within the community is clear about the behaviour that is expected of them, and that newcomers know what kind of environment they can expect to find here.
```